docs/introduction/install.rst

```diff
@@ -2,67 +2,54 @@
+.. contents::
+   :local:
 
-We highly recommend using `Conda <https://conda.io/docs/>`__ to manage environments, i.e. use Conda to create an environment with Node.js installed where you can use Auspice. It's possible to use other methods, but this documentation presupposes that you have Conda installed.
+Install dependencies
+====================
 
-To run package scripts, the `bash shell <https://en.wikipedia.org/wiki/Bash_(Unix_shell)>`__ and the `env <https://en.wikipedia.org/wiki/Env>`__ command need to be in your ``PATH``. You should already have them on Unix-like systems including Linux and macOS. If you are working from Windows, you can run the installation under Git Bash, MSYS2, or Cygwin. You can also use the Windows Subsystem Linux for a fuller Linux environment.
+Auspice is a JavaScript program, and requires `Node.js <https://nodejs.org/>`__ to be installed on your system. Refer to ``engines.node`` in `package.json <https://github.com/nextstrain/auspice/blob/-/package.json>`__ for currently supported versions.
 
-Create a Conda Environment
-==========================
+We recommend using `Mamba <https://mamba.readthedocs.io/>`__ to create an environment with a specific version of Node.js. It's possible to use other methods, but these are the instructions for Mamba:
 
 .. code:: bash
 
-   conda create --name auspice nodejs=14
+   mamba create -c conda-forge --name auspice nodejs=14
    conda activate auspice
 
-..
-
-   This parallels `the Nextstrain installation docs <https://nextstrain.org/docs/getting-started/local-installation#install-augur--auspice-with-conda-recommended>`__. You're welcome to use those instead!
-
-Install Auspice from npm
-========================
+Install Auspice as a user
+=========================
 
 .. code:: bash
 
    npm install --global auspice
 
-Auspice should now be available as a command-line program - check by running ``auspice --help``.
-
 If you look at the `release notes <https://docs.nextstrain.org/projects/auspice/en/stable/releases/changelog.html>`__ you can see the changes that have been made to Auspice (see your version of Auspice via ``auspice --version``). To upgrade, you can run
 
 .. code:: bash
 
    npm update --global auspice
 
-Installing from Source
-======================
+Install Auspice as a developer
+==============================
 
-This is useful for debugging, modifying the source code, or using an unpublished feature branch. We're going to assume that you have used Conda to install Node.js as above.
+This is useful for debugging, modifying the source code, or using an unpublished feature branch.
 
 .. code:: bash
 
-   # activate the correct conda enviornment
-   conda activate auspice
-
    # grab the GitHub auspice repo
    git clone https://github.com/nextstrain/auspice.git
    cd auspice
 
    # install dependencies and make `auspice` available globally
    npm install --global .
```

Comment:
Why suggest installing (really, "linking" in npm terms) the development version instead of directly running `./auspice.js` from the git repo?

(I realize you didn't write this bit, @victorlin, but asking since we're updating this doc anyway. Maybe @jameshadfield has insight here?)

Reply:
> it allows you to use the `auspice` command from any directory you're in

> I see `npm install --global .` as analogous to `python3 -m pip install -e '.[dev]'` for augur.

These are good points, and I'll keep as-is unless there are any alternative suggestions.

Separate but related, #1527 proposes using `npx` to interface with Auspice CLI as a user (not developer).

Reply:
Yeah, that's makes sense since it's gonna be behind a `conda activate` anyway.